src/index.html

```diff
@@ -10,7 +10,33 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="post" class="form-field">
+      <label class="bigInput">
+        <img src="./images/Search.svg" 
+          alt="search" class="icon"
+        >
+        <input 
+          data-qa="big"
+          class="input"
+          type="text" 
+          name="place"
+          placeholder="Try &quot;Los Angeles&quot;"
+          required  
+        >
+      </label>
+      <label class="smallInput">
```

Comment:
add a loop to the second input as well
<img width="280" alt="Screenshot 2023-04-04 at 20 55 53" src="https://user-images.githubusercontent.com/117587463/229877804-ba852889-8d90-4a5c-a9db-e48ff304ada4.png">

Reply:
What did you mean by this task?